src/typescript/sdk/src/indexer-v2/types/json-types.ts

```diff
@@ -365,6 +366,7 @@ export type DatabaseJsonType = {
       ProcessedFields &
       UserLPCoinBalance & { daily_volume: Uint128String }
   >;
+  [DatabaseRpc.RandomSymbols]: { emojis: string };
```

Comment:
```suggestion
  [DatabaseRpc.RandomSymbols]: { emojis: `\x${string}` };
```

Let's update this if this is the case- not sure if it actually is though.